packages/blockchain-wallet-v4/src/utils/btc.js

```diff
@@ -214,7 +215,7 @@ export const calculateBalanceSatoshi = (coins, feePerByte) => {
 }
 
 export const isKey = function (btcKey) {
-  return btcKey instanceof ECPair
+  return btcKey instanceof Object // hack since ECPair instance of is breaking
```

Comment:
How does it break?

Reply:
For some reason the instanceof ECPair equality is false after upgrade. Will investigate further.

Reply:
https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/ecpair.js

<img width="533" alt="Screen Shot 2019-11-04 at 4 04 26 PM" src="https://user-images.githubusercontent.com/55096802/68162708-a4c93280-ff26-11e9-9e06-cf0f384a297f.png">

This breaking makes sense since the the new ecpair export is really just a set of functions and not the ECPair class itself. 

https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/ecpair.js

Will look for a better way to check for ECPair class.

Reply:
```
export const isKey = btcKey => {
  const mockKey = ECPair.fromPrivateKey(crypto.sha256('mock privatekey'))
  return btcKey.constructor === mockKey.constructor
}
```